### ISSUE TITLE
Webpack error - critical dependency the request of a dependency is an expression

### ISSUE THREAD

**Issue by aorumbayev** [algorand-devrel/beaker-ts#25]
I have a simple typescript + next.js setup relying on webpack.

When opening a page that contains references to imports from the generated ts class that beaker-ts produces, i get the following:

```
./node_modules/typescript/lib/typescript.js
Critical dependency: the request of a dependency is an expression

Import trace for requested module:
./node_modules/typescript/lib/typescript.js
./node_modules/beaker-ts/lib/generate/generate.js
./node_modules/beaker-ts/lib/generate/index.js
./node_modules/beaker-ts/lib/index.js
./contracts/helloWorld.ts
Window object is not available, skipping initialization.
```

Seems like there may be either some lazy loading going or it's webpack specific bug. 

This doesn't affect the functionality though and it still works but wondering if there may be some hints how to resolve this warning.

**Comment by barnjamin** [comment]
weird! I've not used next.js before but I can try to use it for the next demo.

It probably doesn't help much but here is a vite demo https://github.com/algorand-devrel/hello-beaker

**Comment by aorumbayev** [comment]
yeah i suspect it has to do something with webpack specifically, a bunch of forums with similar errors. I'll try to post a sandbox that replicates this error once i get some time

**Comment by aorumbayev** [comment]
i think this is the related bug in typescript itself :/ 
https://github.com/microsoft/TypeScript/issues/39436

**Comment by aorumbayev** [comment]
@barnjamin found a way to at least suppress it for now i guess could serve as a temporary hotfix as i haven't found this affecting anything:

```
    webpack: (config, options) => {
        config.module.noParse = [
            require.resolve("typescript/lib/typescript.js"),
        ];
        return config;
    }
```
